apis/stardust/types/index.d.ts

```diff
@@ -795,8 +795,15 @@ declare namespace stardust {
         hypercubePath: string;
     })=>stardust.ExportFormat;
 
+    type onPropertyChange = (properties: qix.GenericObjectProperties)=>void;
+
+    interface QAEProperties {
+        initial?: qix.GenericObjectProperties;
+        onChange?: stardust.onPropertyChange;
+    }

```

Comment:
Is this correct? Seems to diff between charts:
https://github.com/qlik-trial/sn-kpi/blob/master/src/utils/qae.js#L6
https://github.com/qlik-trial/nebula-tables/blob/main/packages/sn-table/src/qae/index.ts#L8
Nebula seems to want the "initial" part, but a bit unsure.

Reply:
looked again. and it looks to support both options
https://github.com/qlik-oss/nebula.js/blob/286be6bebff2aede3e29ffcdb5589f72a58e67b2/apis/supernova/src/qae.js#L125-L130